assets/airports/eddf.json

```diff
@@ -6,940 +6,5425 @@
+            ]
+        },
+        "TOBAK3R": {
+            "icao": "TOBAK3R",
+            "name": "Tobak Three Romeo",
+            "rwy": {
+                "EDDF18": []
+            },
+            "body": [["DF158", "A08+|S220-"], ["DF159", "A25+"], "DF168", "TUKRU", "MTR"],
+            "exitPoints": {
+                "TOBAK": ["TOBAK"]
+            },
+            "draw": [
+                ["DF158", "DF159", "DF168", "TUKRU", "MTR", "TOBAK"]
+            ]
+        },
+        "TOBAK4H": {
+            "icao": "TOBAK4H",
+            "name": "Tobak Four Hotel",
+            "rwy": {
+                "EDDF25L": []
+            },
+            "body": [["^DF980", "A08+"], ["VFM", "S200-"], ["DF180", "A25+"], "ROXAP", ["ADEVO", "A60+"], "LISKU", "TABUM", "TESGA"],
+            "exitPoints": {
+                "TOBAK": ["TOBAK"]
+            },
+            "draw": [
+                ["DF980", "VFM", "DF180", "ROXAP", "ADEVO", "LISKU", "TABUM", "TESGA", "TOBAK"]
+            ]
+        },
+        "TOBAK5F": {
+            "icao": "TOBAK5F",
+            "name": "Tobak Five Foxtrot",
+            "rwy": {
+                "EDDF25L": [["DF235", "A08+"]],
+                "EDDF25C": [["DF234", "A08+"]]
+            },
+            "body": [["DF233", "A35+"], ["DF132", "A44+"], "TABUM", "TESGA"],
+            "exitPoints": {
+                "TOBAK": ["TOBAK"]
+            },
+            "draw": [
+                ["DF235", "DF233"],
+                ["DF234", "DF233", "DF132", "TABUM", "tESGA", "TOBAK"]
+            ]
+        },
+        "TOBAK6M": {
+            "icao": "TOBAK6M",
+            "name": "Tobak Six Mike",
+            "rwy": {
+                "EDDF25C": []
+            },
+            "body": [["^DF999", "A08+"], ["VFM", "S200-"], ["DF180", "A25+"], "ROXAP", ["ADEVO", "A60+"], "LISKU", "TABUM", "TESGA"],
+            "exitPoints": {
+                "TOBAK": ["TOBAK"]
+            },
+            "draw": [
+                ["DF999", "VFM", "DF180", "ROXAP", "ADEVO", "LISKU", "TABUM", "TESGA", "TOBAK"]
+            ]
+        },
+        "TOBAK6T": {
+            "icao": "TOBAK6T",
+            "name": "Tobak Six Tango",
+            "rwy": {
+                "EDDF18": []
+            },
+            "body": [["DF160", "A08+|S220-"], ["DF200", "S250-"], ["PIPIX", "A90+|S250-"], ["VETUX", "A100"], "RUDUS", "MABOB", "TESGA"],
+            "exitPoints": {
+                "TOBAK": ["TOBAK"]
+            },
+            "draw": [
+                ["DF160", "DF200", "PIPIX", "VETUX", "RUDUS", "MABOB", "TESGA", "TOBAK"]
+            ]
+        },
+        "TOBAK7G": {
+            "icao": "TOBAK7G",
+            "name": "Tobak Seven Golf",
+            "rwy": {
+                "EDDF25L": [["DF235", "A08+"]],
+                "EDDF25C": [["DF234", "A08+"]]
+            },
+            "body": ["DF133", ["^DF236", "A35+"], ["DF238", "A44+"], "TABUM", "TESGA"],
+            "exitPoints": {
+    
```

Comment:
All arrivals are expecting 25R, would it make more sense to give some aircraft 25L instead?